eclair-node/src/test/scala/fr/acinq/eclair/api/ApiServiceSpec.scala

```diff
@@ -1168,6 +1170,18 @@ class ApiServiceSpec extends AnyFunSuite with ScalatestRouteTest with IdiomaticM
         assert(serialization.write(chcl) === expectedSerializedChcl)
         system.eventStream.publish(chcl)
         wsClient.expectMessage(expectedSerializedChcl)
+
+        val msgrcv = OnionMessages.ReceiveMessage(MessageOnion.FinalPayload(TlvStream[OnionMessagePayloadTlv](
+          Seq(
+            OnionMessagePayloadTlv.EncryptedData(ByteVector.empty),
+            OnionMessagePayloadTlv.ReplyPath(Sphinx.RouteBlinding.create(PrivateKey(hex"414141414141414141414141414141414141414141414141414141414141414101"), Seq(bobNodeId), Seq(hex"000000")))
+          ), Seq(
+            GenericTlv(UInt64(5), hex"1111")
+          ))), None)
```

Comment:
Can you add a `path_id` so that we test its serialization as well?

Reply:
Done